PokeAlarm/Events/QuestEvent.py

```diff
@@ -38,28 +39,43 @@ def __init__(self, data):
         self.custom_dts = {}
 
         # Quest Details
-        self.quest = data['quest']
-        self.reward = data['reward']
-        # To ensure Windows compatibility, epoch calculation is required
-        self.expire_time = datetime.utcfromtimestamp(
-            data.get(
-                'expire_time',
-                (datetime.combine(date.today(), time(23, 59)) -
-                 datetime(1970, 1, 1)).total_seconds()))
-        self.reward_type = check_for_none(int, data.get('type'), 0)
+        self.quest_type_raw = data['quest_type']
```

Comment:
Are all these fields requires required? right now it'll throw an error for any that don't exist

Reply:
These should probably be .gets for backwards compatibility